### PR TITLE
Added link/connections to marshmallow

### PR DESCRIPTION
**Pull request by avpeery**
Simple setup with marshmallow - added to requirements.txt, set up marshmallow schema in models/meter.py on for the meter model, set up connection, added the import statements on model/models.py and api/models.py

### CODE REVIEW

**Comment by avpeery** [comment]
@jongoncalves  Set up for marshmallow is completed. Came across a couple bugs along the way that I fixed. Let me know if this looks ok to merge! Thanks

File: web/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+marshmallow-sqlalchemy==0.22.3
+pycparser==2.20
 PyMySQL==0.9.3
-cryptography==2.9.2
\ No newline at end of file
+python-dateutil==2.8.1
+python-editor==1.0.4
+six==1.14.0
+SQLAlchemy==1.3.16
+Werkzeug==1.0.1
```
Comment (jongoncalves) [inline web/requirements.txt:24]: Why did you add all of these explicitly? Most are sub-dependencies of the libraries already specified.

File: web/README.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ mysql -u root -p 'your root password'
 
 ## Almost there...
 ```bash
+# set the flask environmental variable
+export FLASK_APP=web
```
Comment (jongoncalves) [inline web/README.md:40]: Nice catch

**Comment by avpeery** [comment]
Yep - I am planning on implementing those changes in a different branch

File: web/requirements.txt (hunk 1 above)
Comment (avpeery) [reply web/requirements.txt:24]: I used pip freeze > requirements.txt to catch what to install for marshmallow but I can edit it to the shorten version if it is preferred

**Comment by jongoncalves** [comment]
> Yep - I am planning on implementing those changes in a different branch

got it, cool

File: web/requirements.txt (hunk 1 above)
Comment (jongoncalves) [reply web/requirements.txt:24]: since the sub dependencies get auto installed, let's leave them out. Just add the marshmallow here and it should be good.

Comment (avpeery) [reply web/requirements.txt]: Ok I updated requirements.txt!

**Comment by avpeery** [comment]
@jongoncalves should I go ahead and merge? or will you? Thanks!
